src/components/auth/magic/login-magic-signup.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div>
+    <v-card-text>
+      <p>
+        We need a few more details to create your account for
+        <i>{{ email }}</i>.
+      </p>
+      <v-text-field
+        v-model.trim="firstName"
+        :error-messages="firstNameErrors"
+        label="First Name"
+        placeholder="Jo"
+        outlined
+        @blur="$v.firstName.$touch()"
+      />
+      <v-text-field
+        v-model.trim="lastName"
+        :error-messages="lastNameErrors"
+        label="Last Name"
+        placeholder="Smith"
+        outlined
+        @blur="$v.lastName.$touch()"
+      />
```

Comment:
We actually *might* be able to pull this info from the UWA directory. No harm in asking though

Reply:
idk, the goal was to remove dependencies on UWA systems though. And the name isn't super important.